ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main__lightspeed.yaml

```diff
@@ -40,7 +40,7 @@ tests:
         export WISDOM_SECRET_PATH=/var/run/psap-wisdom
         export WISDOM_PROTOS_SECRET_PATH=/var/run/psap-wisdom-protos
 
-        run ansible-llm load_test prepare
+        run lightspeed test prepare
```

Comment:
can you `prepare_ci` and `test_ci` entrypoint names?

the idea behind the `_ci` suffix is that `prepare` and `test` should be runnable from the laptop, 
while the `*_ci` entrypoint are reserved for any CI-specific things that should be done before

Reply:
Makes sense, done.